privacy-security-explainer.md

```diff
@@ -117,10 +118,10 @@ function onSessionRafCallback(XRFrame frame) {
 }
 ```
 
-As a result, when more than one view is present, the UA must additionally ensure that:
+In addition to meeting the [`XRPose`](#xrpose) requirements, every call to `XRFrame.getViewerPose()` which will return more than one `XRView` must additionally ensure that:
 * User intention is well understood, either via [explicit consent](#explicit-consent) or [implicit consent](#implict-consent)
-* If the device supports configurable or factory-calibrated interpupillary distance that may vary from device to device, then the XRView transform data must be rounded to prevent fingerprinting. Specific precision for rounding is at the discretion of the user agent.
-* If the XRViewerPose will include multiple XRViews for displays whose positions and orientations have been configured by the user (e.g. in a CAVE) then the XRView transform data must be rounded to prevent fingerprinting.
+* If `XRView` data is affected by settings that may vary from device to device, such as static interpupillary distance or variations in screen geometry, then the XRView data must be anonymized to prevent fingerprinting. Specific approaches to this are at the discretion of the user agent.
```

Comment:
Anonymization only prevents fingerprinting (not user profiling). The profiling threat is addressed through user consent. To clarify the actual profiling threat, we should add text and I've added #748 for that.

In the meantime, this line and the following could probably be combined. Suggested change for this line (and the following one removed):

```suggestion
* If `XRView` data is affected by settings that may vary from device to device, such as static interpupillary distance, variations in screen geometry, or user-configured interpupillary distance, then the XRView data must be anonymized to prevent fingerprinting. Specific approaches to this are at the discretion of the user agent.
```

Reply:
I'm a bit confused.  I separated these lines because I thought we needed to categorize the profiling threat separately from the fingerprinting threat.  Is that not the case?

Reply:
We do, but anonymization (e.g. rounding) only addresses the fingerprinting risk. We can collapse all the fingerprinting mitigations into  a single line since they're the same.

For profiling risk, the mitigation is informed user consent. I've got suggested text in a comment for #748

Reply:
Ah, ok, so this is a bit tricky, because the informed user consent would need to be requested at the time the reference space is created not at the time pose data is queried.  (That's actually one of the reasons I didn't just go ahead and merge your proposed changes in #748 into this PR).  I can collapse these two sentences for now, but we'll need to make sure any additional text regarding consent is added to the correct place in the document.  Sound good?

Reply:
Actually... what about this:
>* If `XRView` data is affected by a user-configured interpupillary distance, then it is strongly recommended that explicit consent be requested for the creation of the `XRReferenceSpace` passed into `XRFrame.getViewerPose()`.

Reply:
SGTM. Can you outline your concerns about what could be tricky in #748?